### ISSUE TITLE
No scripts folder

### ISSUE THREAD

**Issue by mrstretch** [gh0stzk/dotfiles#283]
Hello I have just installed the dotfiles but in the .config/bspwm I do not have the scripts folder.

**Comment by gh0stzk** [comment]
check if there is a src script. Must be ~/.config/bspwm/src

In past commits i changed lots of files, that point to .../.config/bspwm/scripts, to the new src directory. Maybe you have an old files.

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/bf2828af-5d61-44f4-a3c7-635cd52cf37c)

**Comment by gh0stzk** [comment]
yes you have the src directory, something doesn't work?

**Comment by mrstretch** [comment]
i can't click on the wifi icon to see my network

**Comment by gh0stzk** [comment]
Open your filemanager, go to ~/.config/bspwm/src/ and search for NetManagerDM file, i use thunar, in thunar, right click, properties, then in permissions tab, check if the file have execute permissions, if not, give it the execute permissions.

![Shot-2024-08-05-161609](https://github.com/user-attachments/assets/1c398819-3ce8-4b0c-8b73-b01e691d5c5d)


If that doesnt work, open a terminal and write NetManagerDM, check what error appear in terminal.

**Comment by gh0stzk** [comment]
And You need to use NetworkManager as your connection manager. if you doesnt use networkmanager, thats the problem, the applet search for networkmanager.

**Comment by mrstretch** [comment]
what’s your file manager please ?

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/f5774bcb-5340-425f-a8db-e55e69317b7c)

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/40ba1d81-077d-4320-b65b-5e1bcd75edd2)

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/05fb01f8-eb66-4c5b-ac9a-454105dc531a)

**Comment by gh0stzk** [comment]
all right, I see that you use a wired connection, which is probably why the internet works for you without having a network manager.

And I see that NetworkManager is not active, that is the problem.

Just activate networkmanager

`sudo systemctl enable NetworkManager.service`

Reboot and your network should appear in the applet.

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/147751e7-db18-4539-baa2-5258a0840439)
when I want to create a new file on neovim
avec le screen

**Comment by mrstretch** [comment]
networkmanager it’s ok

**Comment by mrstretch** [comment]
do you have discord ? or a support server 
it’s more easier for me

**Comment by gh0stzk** [comment]
press enter in neovim, then do `:Lazy` you will see the plugin manager, press `Shift + U` to update all plugins and that fix the problem.

**Comment by mrstretch** [comment]
neovim it’s fixed ty
 I just downloaded an mp3 in Music but when I do shift alt k I don't see it.

**Comment by gh0stzk** [comment]
in ncmpcpp which is `shift + alt + k` press the key numer ` 2` then press `u` should appear database updated, close it, open again and should work now.

**Comment by gh0stzk** [comment]
Be sure the directory where your music is stored is the same dir that apper in mpd.conf

**Comment by gh0stzk** [comment]
https://github.com/gh0stzk/dotfiles/wiki/MPD-Configuration

**Comment by mrstretch** [comment]
it's possible to see our music from the polybar instead of use shift + alt + k

**Comment by mrstretch** [comment]
how to get the 24-hour format on polybar?

**Comment by gh0stzk** [comment]
> it's possible to see our music from the polybar instead of use shift + alt + k

if you already can play music from mpd, the polybar module should show the name of the current song and  a control buttoms. not in all of my rices show the song name, for example in emilia the song name is showed  and the control meda buttoms.

If they doesnt show in your polybar, maybe there is an error somewhere. kill polybar, and open a terminal, and for example for emilia rice, paste this; 

`polybar --log=warning emi-bar -c $HOME/.config/bspwm/rices/emilia/config.ini`

and see if are errors.

**Comment by gh0stzk** [comment]
> how to get the 24-hour format on polybar?

edit modules.ini file inside every rice directory, search for the date module and edit:

`time = %I:%M %P` 

to this

`time = %H:%M`

**Comment by mrstretch** [comment]
![image](https://github.com/user-attachments/assets/d6a9ca13-107a-465d-9e8d-a3aac621783b)
what’s the path to chnage the time ? 
and for the music it’s fixed

**Comment by mrstretch** [comment]
how can i put your filemanager theme ?

**Comment by gh0stzk** [comment]
To set a theme for gtk applications, open the app menu, search for lxappearance, i dont remember how is it in menu, because i have ir in spanish, but something like **Customize Look and Feel** then, if you installed a them via pacman, the theme will be shown there. If you wanna download from internet, put the theme in ~/.themes folder, you need to create the folder. and the theme will be show in the app.

**Comment by mrstretch** [comment]
what’s the thème you are using in your videos ?

**Comment by gh0stzk** [comment]
I use dracula theme from https://draculatheme.com/gtk

**Comment by mrstretch** [comment]
it’s possible to custom the login page ?
it’s possible to add a thunar theme for the next rice?

**Comment by gh0stzk** [comment]
> it’s possible to custom the login page ? it’s possible to add a thunar theme for the next rice?

Yes. and the second question is, About the gtk themes, it would be the only functionality that would be missing in my dotfiles so that everything changes when you change the theme, as I have explained before, I have not found a theme, simple, pleasant and easy to edit to make 18 themes. I already found one but I need to test it first and if everything is fine, then I have to convert it to the 18 themes, so for the moment that won't happen.
